src/server/scripts/EasternKingdoms/ZulAman/zulaman.cpp

```diff
@@ -201,14 +202,14 @@ struct npc_forest_frog : public ScriptedAI
         events.ScheduleEvent(eventTimer, 3000);
 
         me->UpdateEntry(cEntry);
+        me->SetFacingToObject(PlayerCaster());
```

Comment:
this introduces nullptr crash if player disconnects or leaves map

Reply:
does removing “GetMap()” solve the problem?

Reply:
No, you need to check if the player object is still valid before calling SetFacingToObject()

Reply:
In this case I have to validate using if?

Reply:
Correct